src/main/java/marquez/api/resources/DatasourceResource.java

```diff
@@ -62,6 +65,29 @@ public Response list(
     return Response.ok(DatasourceResponseMapper.toDatasourcesResponse(datasources)).build();
   }
 
+  @GET
+  @ResponseMetered
+  @ExceptionMetered
+  @Timed
+  @Path("/{datasourceId}")
```

Comment:
We discussed returning a datasource by URN, has that changed? see #352

Reply:
You're right. Will fix that.

Reply:
Great. Let's also remove the class `DatasourceId`

Reply:
I've addressed this in the next PR, #394.

Reply:
with the stacked PR approach, its usually best to keep all of the changes to a single file within the same PR in the stack. Addressing the feedback from one PR in the stack in a different PR is not ideal.

Reply:
I think we should back off from stacked PRs for now, it seems to be bringing more confusion than clarity

Reply:
+1